### PR TITLE
Add header and footer to HTMLs

### PR DESCRIPTION
**Pull request by cpotter88**
Added header and footer elements to HTML files.

### CODE REVIEW

**Review by Nebbs82 — COMMENTED** [review]
Aces

File: assets/ContactPage.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (MrAwesome2127) [inline assets/ContactPage.html:4]: Please add Header/Footer to this PR on this page as well.

**Review by MrAwesome2127 — APPROVED** [review]
g2g

**Review by Nebbs82 — COMMENTED** [review]
Yup

**Review by Nebbs82 — APPROVED** [review]

**Pull request merged**